tools/scripts/test_remote_runner_multi.sh

```diff
@@ -1,25 +1,27 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 BUILD_DIR=$1
+TEST_BINARY=$2
+shift 2
 
-PORT1=$((3000 + $RANDOM % 6000))
-PORT2=$(($PORT1 + 10))
+PORT1=11000
```

Comment:
Is this better than random?

Reply:
We can't guarantee there won't be collisions with random anyway, and this should make errors more deterministic and therefore easier to spot.